tool/gravity/cli/config.go

```diff
@@ -335,6 +335,11 @@ func (i *InstallConfig) CheckAndSetDefaults() (err error) {
 	if err := checkLocalAddr(i.AdvertiseAddr); err != nil {
 		return trace.Wrap(err)
 	}
+
+	if err := checkOverlayNet(i.AdvertiseAddr, i.ServiceCIDR, i.PodCIDR); err != nil {
```

Comment:
These command line options is not the only way to configure service/pod subnets - ClusterConfiguration resource is another way to set these up.
During installation these options get merged with an optional cluster configuration resource which then gets validated as a whole. I now spotted a problem that the [PR](https://github.com/gravitational/gravity/pull/1975/) which introduced these changes was not ported to master which I would do before doing these changes.

Reply:
I submitted https://github.com/gravitational/gravity/pull/2208 to backport changes and I'd like this PR to go after.